packages/react-tooltip/src/components/Tooltip/Tooltip.tsx

```diff
@@ -3,12 +3,11 @@ import { useTooltip_unstable } from './useTooltip';
 import { renderTooltip_unstable } from './renderTooltip';
 import { useTooltipStyles_unstable } from './useTooltipStyles';
 import type { TooltipProps } from './Tooltip.types';
-import type { ForwardRefComponent } from '@fluentui/react-utilities';
 
 /**
  * A tooltip provides light weight contextual information on top of its target element.
  */
-export const Tooltip: ForwardRefComponent<TooltipProps> = React.forwardRef((props, ref) => {
+export const Tooltip = React.forwardRef<HTMLElement, TooltipProps>((props, ref) => {
```

Comment:
There's no ref to forward, since there's no root slot.
```suggestion
export const Tooltip: React.FC<TooltipProps> = props => {
```

Reply:
Done in 61307b6